packages/api-clients/open-api/bffApi.yml

```diff
@@ -12721,6 +12721,105 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  "/delegations/{delegationId}/contracts/{contractId}":
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+      - name: delegationId
+        in: path
+        schema:
+          type: string
+          format: uuid
+        required: true
+      - name: contractId
+        in: path
+        schema:
+          type: string
+          format: uuid
+        required: true
+    get:
+      tags:
+        - delegations
+      summary: Retrieve a contract of a delegation
+      description: Retrieve a contract of a delegation
+      operationId: getDelegationContract
+      responses:
+        "200":
+          description: Contract retrieved
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          headers:
+            "X-Rate-Limit-Limit":
+              schema:
+                type: integer
+              description: Max allowed requests within time interval
+            "X-Rate-Limit-Remaining":
+              schema:
+                type: integer
+              description: Remaining requests within time interval
+            "X-Rate-Limit-Interval":
+              schema:
+                type: integer
+              description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+        "400":
```

Comment:
This error is never raised in the code so we can delete it; meanwhile the delegation process also returns `401` that should be added here too